### ISSUE TITLE
Forward slashes in scheduler times cause the server to crash

### ISSUE THREAD

**Issue by rezafouladian** [bbilly1/tubearchivist#126]
When configuring the scheduler, it does not accept values such as `*/5` (every 5 minutes in cron) but will accept values such as `0/5` (every 5 minutes starting a minute 0).'

This results in the server failing to start with errors such as:
```
ValueError: invalid literal for int() with base 10: '0/5'
```
```
ValueError: Invalid weekday literal '0/5'.
```

It also uses term "weekday literal" even though the value set was for minutes.

**Comment by bbilly1** [comment]
You are right, so that's partially by design, to avoid people spamming youtube and getting themself blocker by for example refreshing their subscriptions every minute, hence the "no wildcard" thing [here](https://github.com/bbilly1/tubearchivist/wiki/Settings#scheduler-setup), but I didn't account for the 0/5 notation, only for the * notation. I think there is another lower level safeguard from the scheduler in there, that produces the confusing *weekday* message.

So I think the correct validation for the minute field here should be to only allow numbers and no letters.

Just in case you need to recover your installation, you'll have to delete the faulty configuration from redis like so:
```shell
docker exec archivist-redis redis-cli JSON.DEL ta:config
```
then restart tubearchivist:
```shell
docker restart tubearchivist
```

This will reset things to default, so make sure to at least set your download settings before starting to download again.

Thank you for bringing this to my attention.

**Comment by bbilly1** [comment]
OK, I think that should do it. Thank you for your help! That's now merged in v0.1.0
